index.js

```diff
@@ -61,4 +62,5 @@ app.listen(port, () => {
 	Logger.debug(`Vulx initialized on port ${port}`);
 	if(process.pkg)
 		open(link);
-});
\ No newline at end of file
+});
+module.exports = app;
```

Comment:
Not sure what this is needed for, I assume its for 'expressHelper'
If so, let me know.